### PR TITLE
Edited FigureElement.scale() to handle x and y directions correctly. Created an example too.

### PR DESCRIPTION
**Pull request by shervinsahba**
I had some issues with the `scale_xy()` function and noticed another user had the same issue (#28). I deleted the method for `compose.Element.scale()` since it seemed redundant and edited the `scale_xy()` function for the parent class `transform.FigureElement` to be the new `scale()` function. It appears to now scale both directions, and I don't think it changes how the images translate.

Check out the new `examples/compose_scaling.py`! 

Aside: I had to add a new image, `files/examples/Example.svg`, for the example since `svg_logo.svg` throws errors for me, even with the pre-existing examples. I believe it has to do with the height and width being "100%" ([see line 3 here](https://github.com/btel/svg_utils/blob/master/examples/files/svg_logo.svg?short_path=6acae67)), and the `Units` methods currently not knowing how to handle that.

Thanks!

### CODE REVIEW

**Review by btel — APPROVED** [review]
thanks for your contribution, it looks great

I am aware of the error due to height/weight of the image being set in percent. I added a test for it in PR #54, but still did not implement the test.

File: src/svgutils/transform.py [hunk 1]
```diff
@@ -104,9 +108,12 @@ def scale_xy(self, x=0, y=None):
             y-axis scaling factor. To scale down y < 1, scale up y > 1.
 
         """
-        self.root.set("transform", "%s scale(%f %f)" %
-                      (self.root.get("transform") or '',
-                       x, y if y is not None else ''))
+        # self.root.set("transform", "%s scale(%f %f)" %
+        #               (self.root.get("transform") or '',
+        #                x, y if y is not None else ''))
```
Comment (btel) [inline src/svgutils/transform.py:113]: you can remove these comments

```suggestion
```

**Comment by btel** [comment]
I just merged the fix for the "100%" height/width. I also fixed an issue with the CI, so the CI status for the PR should appear on the PR page. Could you just remove the commented lines from the code? You can simply apply the suggestion that I inserted (or do it locally from your computer and just push a new commit).

**Comment by shervinsahba** [comment]
Done! Thanks.

**Comment by btel** [comment]
wonderful! Thanks a lot!

**Pull request merged**